src/tabs/connect/Connect.js

```diff
@@ -30,7 +34,19 @@ class Connect extends PureComponent {
           <ScrollView>
             <UserAvatarHeader />
             <ActionBar />
-            <HorizontalLikedContentFeedConnected />
+            <HorizontalLikedContentFeedConnected
+              Component={({ ...feedProps }) => (
+                <HorizontalLikedContentFeed
+                  Component={({ ...cardProps }) => (
+                    <HorizontalContentCardConnected
+                      labelText={''}
+                      {...cardProps}
+                    />
+                  )}
+                  {...feedProps}
+                />
+              )}
+            />
```

Comment:
I _hate_ that we have to do this just to remove the `labelText`. Maybe a change to core they might consider is to be able to use the default `Component` but still be allowed to override props on that component?

Reply:
I tried